xml/art_sle_ha_install_quick.xml

```diff
@@ -641,7 +652,7 @@ softdog                16384  1</screen>
   </para>
   <para>
    The bootstrap scripts take care of changing the configuration specific to
-   a two-node cluster, for example, SBD and &corosync;.
+   a two-node cluster, for example, SBD, &corosync;, and (optionally) QDevice.
```

Comment:
Basically yes.

There is a reason why this is so foggy: bsc#1171385 and if the bootstrap script asks in a qdevice stage. However, that is a future issue.

I thought it would be easier for now to describe only a scenario that a QDevice is configured _after_ the cluster is already set up. Although I'm aware that it is entirely possible to configure QDevice during the first init phase.

When the above bug/feature is solved, I would cover that in the doc of course. 

Maybe I need to rephrase that. Or leave out QDevice.

Reply:
I mean if we'd like to mention the optional QDevice, the description should be put into `init` section. Since we cannot set up QDevice with `join` anyway.

Reply:
Yes, you're right. I removed it.

After a second thought, I think, we should leave out QDevice for the moment. There is a small list item about QDevice/QNetd which basically refers to the new chapter. IMHO, that is currently enough. Once the feature about the crm shell is settled and implemented, we can revisit this and integrate that into the quick start (if feasible). That would be my plan. :wink: